crates/subspace-service/src/mmr.rs

```diff
@@ -0,0 +1,9 @@
+use sp_mmr_primitives::utils::NodesUtils;
+use sp_mmr_primitives::{NodeIndex, INDEXING_PREFIX};
+
+pub(crate) mod request_handler;
+pub(crate) mod sync;
+
+pub(crate) fn get_offchain_key(index: NodeIndex) -> Vec<u8> {
+    NodesUtils::node_canon_offchain_key(INDEXING_PREFIX, index)
+}
```

Comment:
Seems like we are adding one kv per block (where the kv is the leaf node), but the MMR storage also contains the intermediate nodes, see https://github.com/paritytech/polkadot-sdk/blob/4aa29a41cf731b8181f03168240e8dedb2adfa7a/substrate/client/merkle-mountain-range/src/offchain_mmr.rs#L176-L178

there are `Vec<NodeIndex>` added by one leaf node.